RibonDAO/core-api
# (TECH-100): Stripe refund execution

# Description

- Create process of refund to manager from stripe

# Necessary changes

- Create route to refund credit card payment
- Att status of person_payment
- Add tests

## Does this pull request close any open issues?

- Put here the issue that must be closed with this PR
- eg: closes [#TECH-100](https://ribon.atlassian.net/jira/software/c/projects/TECH/boards/26?modal=detail&selectedIssue=TECH-100)

## Type of change

Please delete options that are not relevant.


## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Create a new payment and use the external_id in person_payment table to send with as param in credit_card_refund route.
